packages/@coorpacademy-components/src/atom/input-search/index.js

```diff
@@ -0,0 +1,69 @@
+import React, {useMemo} from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import {
+  NovaCompositionCoorpacademySearch1 as CMSearchIcon,
+  NovaCompositionCoorpacademySearch as SearchIcon,
+  NovaSolidStatusClose as CloseIcon
+} from '@coorpacademy/nova-icons';
+import {get, noop, isEmpty} from 'lodash/fp';
+import Provider from '../provider';
+import style from './style.css';
+
+const Search = (props, context) => {
+  const {value, placeholder, onChange = noop, onClear, onFocus, onBlur, theme = 'default'} = props;
+  const {skin} = context;
+  const dark = get('common.dark', skin);
+  const handleChange = useMemo(() => e => onChange(e.target.value), [onChange]);
+  const cmStyle = classnames(style.coorpmanager);
+  const isDefaultTheme = theme === 'default';
+
+  return (
+    <div className={isDefaultTheme ? style.wrapperSearch : cmStyle}>
+      <label htmlFor="search">
```

Comment:
dans ce cas un `title` est indispensable (" search course ")
```suggestion
      <label htmlFor="search" title={searchButtonTitleRgaa...}>
```

Reply:
👍